### PR TITLE
WIP: Implemented change course role assignment.

### PR DESCRIPTION
**Pull request by waheedahmed**
[ECOM-6207](https://openedx.atlassian.net/browse/ECOM-6207)

**Screenshots:**
screen shot 2016-11-28 at 5 42 29 pm
screen shot 2016-11-28 at 5 44 43 pm
screen shot 2016-11-28 at 5 42 50 pm
screen shot 2016-11-28 at 5 43 02 pm

### CODE REVIEW

File: course_discovery/apps/publisher/urls.py [hunk 1]
```diff
@@ -11,6 +11,11 @@
     url(r'^courses/(?P<pk>\d+)/view/$', views.ReadOnlyView.as_view(), name='publisher_courses_readonly'),
     url(r'^courses/(?P<pk>\d+)/edit/$', views.UpdateCourseView.as_view(), name='publisher_courses_edit'),
     url(
+        r'^courses/(?P<course_id>\d+)/change/role_assignment/$',
```
Comment (bderusha) [inline course_discovery/apps/publisher/urls.py]: `r'^courses/(?P<course_id>\d+)/role_assignments/$',`

File: course_discovery/apps/publisher/views.py [hunk 2]
```diff
@@ -354,3 +382,26 @@ def post(self, request):
...
+        try:
+            course = Course.objects.get(id=course_id)
+            users_default_role = OrganizationUserRole.objects.get(
+                organization=course.organizations.first(), user=request.user
+            )
+            current_users_role = course.course_roles.get(role=users_default_role.role)
+            new_user = User.objects.get(id=user_id)
+            current_users_role.user = new_user
+            current_users_role.save()
+            if not mixins.check_view_permission(request.user, course):
```
Comment (bderusha) [inline course_discovery/apps/publisher/views.py]: Should this be the first thing we check before doing any work?

**Review by bderusha — CHANGES_REQUESTED** [review]
A few minor things, but overall this looks good

**Comment by waheedahmed** [comment]
Base branch merged into master, closing this PR and created against master https://github.com/edx/course-discovery/pull/458.

**Pull request closed without merging**